core/state_processor_test.go

```diff
@@ -71,7 +71,13 @@ func TestStateProcessorErrors(t *testing.T) {
 		},
 		{
 			txs: []*types.Transaction{
-				makeTx(0, common.Address{}, big.NewInt(0), 21000000, nil, nil),
+				makeTx(0, common.Address{}, big.NewInt(0), 2100000000, nil, nil),
```

Comment:
here it was failing at 21000000 on geth, but on goq the error is not the same (like 21001)
I needed to increase the number to have the same error.